package.json

```diff
@@ -44,7 +45,7 @@
     "stylus": "^0.54.5",
     "stylus-loader": "^2.3.1",
     "ts-loader": "^0.8.2",
-    "umd-compat-webpack-plugin": "1.0.2",
+    "umd-compat-webpack-plugin": "1.0.4",
```

Comment:
this needs a later version of the umd-compat plugin, as it was seemingly injecting js into entries that weren't js.